lib/util.js

```diff
@@ -316,7 +319,7 @@ function formatValue(ctx, value, recurseTimes) {
     formatter = formatMap;
   } else {
     // Only create a mirror if the object superficially looks like a Promise.
```

Comment:
What do you mean? A mirror is still only created if it's been determined the object is a promise.